cmd/main.go

```diff
@@ -68,7 +71,14 @@ func main() {
 	var secureMetrics bool
 	var enableHTTP2 bool
 	var tlsOpts []func(*tls.Config)
+	var s3Bucket, s3Region string
+	var s3TimeoutSeconds int
+	var imageDirectory string
 	flag.StringVar(&namespace, "namespace", "giantswarm", "The namespace where node image objects are managed.")
+	flag.StringVar(&s3Bucket, "s3-bucket", "", "The S3 bucket where images are stored.")
+	flag.StringVar(&s3Region, "s3-region", "", "The region where the S3 bucket is located.")
```

Comment:
We could potentially also give the url of the bucket and derive the region and name from it :thinking: